tools/raml2html/themes/default/resource-group.html.twig

```diff
@@ -1,10 +0,0 @@
-{% if resource.resources|length > 1 %}
```

Comment:
If a resource has methods and more than one sub-resources, only its resources are displayed.

- GET a/b (is not skipped as there is one sub-resource)
- GET a/b/c
- GET d/e (is skipped as there is two sub-resources)
- GET d/e/f
- GET d/e/g

Reply:
Without this fix, the following route is skipped: `POST /cart`

- POST /cart (is skipped as there is more than one sub-resource)
- POST /cart/view
- GET /cart/{identifier}
- …